src/logic/logic.shaii.ts

```diff
@@ -427,6 +427,40 @@ export function britify(sentence: string): string {
   return sentence;
 }
 
+/**
+ * Spongify a sentence
+ * @param sentence the sentence to spongify
+ * @param capsOdd if letters to capitalize are the odd ones (by default: true)
+ * @author Qexat
+ */
+export function spongify(sentence: string, capsOdd: boolean = true): string {
+  var newSentence = "";
+  var lastNotSpaceChar;
+
+  if (capsOdd) {
+    newSentence += sentence[0].toUpperCase();
+  } else {
+    newSentence += sentence[0].toLowerCase();
+  }
+  // upper one character out of 2
+  for (var i = 1; i < sentence.length; i++) {
```

Comment:
use let keyword instead here since you don't care for the scope of i leaking out